chocolatey-packages/ipinfo/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,22 @@
+﻿$ErrorActionPreference = 'Stop';
+ 
+$packageName = 'ipinfo'
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$packageArgs = @{
+    packageName    = $packageName
+    url64          = 'https://github.com/ipinfo/cli/releases/download/ipinfo-3.3.0/ipinfo_3.3.0_windows_amd64.zip'
+    checksum64     = '2ab00f6a289d308b9ac04be9153aa566e93acff3da8a62dd283022a0cb6ee34d'
```

Comment:
Will this be auto-updated?

Reply:
yes

Reply:
You mean in this repo itself? How? The app makes a commit or smth?

Reply:
Elaborate

Reply:
No, not in the repo. The choco release workflow runs the update.ps1 which updates this file if a newer release of cli was created. The update.ps1 script also creates a .nupkg file that we then upload to choco. We actually don't need to update this info within the repo.

Reply:
We could add an additional step to the workflow, that creates a PR for update changes.